src/i18n/locale.ts

```diff
@@ -51,6 +51,11 @@ export class Locale {
         return "English"
     }
 
+    //-------------------------------------------------------------------------------------------------------
```

Comment:
I think we waited to not use a function for getting languages. Did we need to do so for some reason ?